src/bout++.cxx

```diff
@@ -698,6 +705,9 @@ void addBuildFlagsToOptions(Options& options) {
   options["use_sigfpe"] = bout::build::use_sigfpe;
   options["use_signal"] = bout::build::use_signal;
   options["use_track"] = bout::build::use_track;
+  options["use_cuda"] = bout::build::use_cuda;
```

Comment:
although I think this should be `has_cuda` ...

Reply:
Yes, that's better